clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/deploy/ops/CreateLambdaAtomicOperation.java

```diff
@@ -69,9 +74,97 @@ private CreateFunctionResult createFunction() {
     request.setCode(code);
     request.setTags(objTag);
 
+    Map<String, String> envVariables = description.getEnvVariables();
+    if (null != envVariables) {
+      request.setEnvironment(new Environment().withVariables(envVariables));
+    }
+
+    if (null != description.getSecurityGroupIds() || null != description.getSubnetIds()) {
```

Comment:
should either of these check for empty as well (subnetIds I suspect should)

alternatively/additionally this would make sense to check in a Validator if it is necessary